core/storage/feedback/gae_models.py

```diff
@@ -420,6 +421,400 @@ def get_multi(cls, user_id, thread_ids):
         return super(FeedbackThreadUserModel, cls).get_multi(instance_ids)
 
 
+class GeneralFeedbackThreadModel(base_models.BaseModel):
+    """Threads for each entity.
+
+    The id of instances of this class has the form
+        [ENTITY_TYPE].[ENTITY_ID].[GENERATED_STRING]
+    """
+    # The type of entity the thread is linked to.
+    entity_type = ndb.StringProperty(required=False, indexed=True)
+    # The ID of the entity the thread is linked to.
+    entity_id = ndb.StringProperty(required=False, indexed=True)
+    # ID of the user who started the thread. This may be None if the feedback
+    # was given anonymously by a learner.
+    original_author_id = ndb.StringProperty(indexed=True)
+    # Latest status of the thread.
+    status = ndb.StringProperty(
+        default=STATUS_CHOICES_OPEN,
+        choices=STATUS_CHOICES,
+        required=True,
+        indexed=True,
+    )
+    # Latest subject of the thread.
+    subject = ndb.StringProperty(indexed=False)
+    # Summary text of the thread.
+    summary = ndb.TextProperty(indexed=False)
+    # Specifies whether this thread has a related learner suggestion.
```

Comment:
Not all suggestions are from learners. Maybe strike "learner".

Reply:
Done